kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -129,15 +141,24 @@ SSLSession sslSession() {
     }
 
     KafkaProxyFrontendHandler(
-                              NetFilter netFilter,
+                              PluginFactoryRegistry pfr,
+                              FilterChainFactory filterChainFactory,
+                              List<NamedFilterDefinition> filters,
```

Comment:
think we need the `Definition` in the name somewhere `filterDefs` or `filterDefinitions`.